airbyte-integrations/connectors/source-webflow/integration_tests/configured_catalog.json

```diff
@@ -0,0 +1,13 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "Blog Authors",
```

Comment:
@sajarin please take a look how other connectors do this part of the `configured_catalog.json`

Reply:
Maybe some other connectors generate dynamically the schema like Salesforce